### PR TITLE
luaver 1.1.0

### PR DESCRIPTION
**Pull request by JaredSartin**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Updates to latest luaver with zsh support

### CODE REVIEW

**Comment by JaredSartin** [comment]
Meant ruby - not rails... #longday

**Comment by JaredSartin** [comment]
For visibility:
```
jared•~» brew test --verbose luaver                                                                                                                                                                                                [15:05:20]
Testing luaver
/usr/bin/sandbox-exec -f /tmp/homebrew20170815-23820-wqfmiu.sb /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby -W0 -I /usr/local/Homebrew/Library/Homebrew -- /usr/local/Homebrew/Library/Homebrew/test.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/luaver.rb --verbose
==> . /usr/local/Cellar/luaver/1.1.0/bin/luaver && luaver install 5.3.3 < /dev/null
Error: luaver: failed
Failed executing: . /usr/local/Cellar/luaver/1.1.0/bin/luaver && luaver install 5.3.3 < /dev/null
/usr/local/Homebrew/Library/Homebrew/formula.rb:1804:in `block in system'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1742:in `open'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1742:in `system'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/luaver.rb:28:in `block (3 levels) in <class:Luaver>'
/usr/local/Homebrew/Library/Homebrew/extend/ENV/std.rb:76:in `deparallelize'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/luaver.rb:28:in `block (2 levels) in <class:Luaver>'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/luaver.rb:27:in `each'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/luaver.rb:27:in `block in <class:Luaver>'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1639:in `block (2 levels) in run_test'
/usr/local/Homebrew/Library/Homebrew/formula.rb:834:in `with_logging'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1638:in `block in run_test'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:14:in `block in mktemp'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:74:in `block in run'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:74:in `chdir'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:74:in `run'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:13:in `mktemp'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1632:in `run_test'
/usr/local/Homebrew/Library/Homebrew/test.rb:28:in `block in <main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/timeout.rb:66:in `timeout'
/usr/local/Homebrew/Library/Homebrew/test.rb:27:in `<main>'
```

**Comment by DhavalKapil** [comment]
`luaver`'s install script no longer sources the profile. I'm not versed with homebrew. Can that be the reason for failing tests?

**Comment by JaredSartin** [comment]
@DhavalKapil maybe... I will revisit the tests

**Comment by JaredSartin** [comment]
@DhavalKapil got it closer - see next commit:

```
==>  lua-5.3.3 successfully installed. Do you want to switch to this version? [Y/n]:
==> /usr/local/Cellar/luaver/1.1.0/bin/luaver use 5.3.3 && lua -v
==>  Switching to lua-5.3.3
==>  Successfully switched to lua-5.3.3
sh: lua: command not found
```

File: Formula/luaver.rb [hunk 1]
```diff
@@ -30,13 +25,14 @@ def caveats; <<-EOS.undent
   test do
     lua_versions = %w[5.3.3 5.2.4 5.1.5]
     lua_versions.each do |v|
-      ENV.deparallelize { system ". #{bin}/luaver && luaver install #{v} < /dev/null" }
-      system ". #{bin}/luaver && luaver use #{v} && lua -v"
+      ENV.deparallelize
```
Comment (ilovezfs) [inline Formula/luaver.rb]: Why is this set unconditionally and no longer scoped to just the installs?

File: Formula/luaver.rb [hunk 2]
```diff
@@ -4,13 +4,8 @@ class Luaver < Formula
...
 
   stable do
-    url "https://github.com/DhavalKapil/luaver/archive/v1.0.0.tar.gz"
-    sha256 "5c7eb65ea9b3cb4f10987c1d564e6e86bd6ad4f9d829a799121c83f0d85bd390"
-
-    patch do
-      url "https://github.com/DhavalKapil/luaver/pull/9.patch?full_index=1"
-      sha256 "0fa92083eabc2e9cb369c3aefbcf17cee15442ad8c22b18cbe6e2a39e226ca8e"
-    end
+    url "https://github.com/DhavalKapil/luaver/archive/v1.1.0.tar.gz"
```
Comment (ilovezfs) [inline Formula/luaver.rb]: The url and checksum should be moved above head and the `stable do ... end` removed.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/luaver.rb (hunk 2 above)
Comment (JaredSartin) [inline Formula/luaver.rb]: Can do

File: Formula/luaver.rb (hunk 1 above)
Comment (JaredSartin) [inline Formula/luaver.rb]: Looks like a leftover of the broken build, will fix

**Comment by ilovezfs** [comment]
Thanks for your first contribution to Homebrew, @JaredSartin! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**